structs/src/strg.rs

```diff
@@ -37,6 +37,68 @@ pub struct Strg<'r>
 
 impl<'r> Strg<'r>
 {
+    fn is_jap_version(languages: Vec<FourCC>) -> bool
+    {
+        languages.len() == 2 && languages.iter().any(|lang| lang.as_bytes() == b"ENGL") && languages.iter().any(|lang| lang.as_bytes() == b"JAPN")
+    }
+
+    pub fn add_strings_2(self: &mut Self, strings: Vec<String>, languages: Vec<FourCC>)
```

Comment:
Please make a single function with `languages: Option<Vec<FourCC>>`, where `None` defaults to 

```rs
vec![
        FourCC::from(b"ENGL"),
        FourCC::from(b"DUTC"),
        FourCC::from(b"FREN"),
        FourCC::from(b"GERM"),
        FourCC::from(b"ITAL"),
        FourCC::from(b"JAPN"),
        FourCC::from(b"SPAN"),
    ]
```